packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1606,14 +1605,9 @@ describe('plugin-meetings', () => {
          to @webex/internal-media-core when addMedia, updateMedia, publishTracks, unpublishTracks are called
          in various combinations.
       */
-      [true,false].forEach((isMultistream) =>
+      [true/false].forEach((isMultistream) =>
```

Comment:
why this change? `true/false` actually evaluates to infinity, which is probably not what we want here....

Reply:
Once again, it should have been a typo from my side. Fixed it and also fixed the tests that failed related to this. 

Done.